examples/sparsestream/stream.py

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import asyncio
+
+import yaml
+
+from deepsparse import Pipeline
+from labels import sentiments, topics
+from rich.console import Console
+from tweepy.asynchronous import AsyncStream
+from usernames import user_id, user_name
+
+
+console = Console(width=100)
+
+
+def get_config(path):
+
+    with open(path) as file:
+        config = yaml.safe_load(file.read())
+
+    return config
+
+
+config_path = "./config.yaml"
+config = get_config(config_path)
+
+sentiment_classifier = Pipeline.create(
+    task=config["task"], model_path=config["sent_model"], scheduler="sync"
```

Comment:
why is task in the config if we know we're going to specifically use it for classification tasks?